ci/appveyor/linux_prepare.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+sudo apt update
+# sudo apt-get upgrade -y
+sudo apt install -y git curl libssl1.0.0 build-essential libsodium23 libsodium-dev autoconf ncurses-dev libssl-dev
+
+PACKAGE_NAME=esl-erlang_${OTP_VERSION}-1~ubuntu~bionic_amd64.deb
+OTP_DOWNLOAD_URL=https://packages.erlang-solutions.com/erlang/esl-erlang/FLAVOUR_1_general/${PACKAGE_NAME}
+curl -fsSL -o /tmp/${PACKAGE_NAME} "$OTP_DOWNLOAD_URL"
+sudo apt remove -y erlang-base
+sudo apt autoremove -y
+sudo apt install -y libwxbase3.0-0v5 libwxgtk3.0-0v5 libsctp1
```

Comment:
Necessary? I am mentioning for making this closer to https://github.com/aeternity/docker-builder/blob/23756220edbca600f3f7af53b0ee7c8f7127cad1/Dockerfile

Reply:
Required because this setup uses the precompiled package from ESL which has these package requirements.